homeassistant/config_entries.py

```diff
@@ -254,6 +258,9 @@ def __init__(self, version: str, domain: str, title: str, data: dict,
         # State of the entry (LOADED, NOT_LOADED)
         self.state = state
 
+        # Listeners to call on update
+        self.update_listeners: list = []
```

Comment:
This is Python 3.6 syntax.

Reply:
Allright, will look into ignore typing for that one then

Reply:
In Python 3.5, define types by adding it as a comment after the declaration.